amanuensis/__init__.py

```diff
@@ -204,12 +205,17 @@ def _setup_arborist_client(app):
     if app.config.get("ARBORIST"):
         app.arborist = ArboristClient(arborist_base_url=config["ARBORIST"])
 
-def _setup_hubspot_key(app):
+def _setup_hubspot_client(app):
     if app.config.get("HUBSPOT"):
-        if "API_KEY" in config["HUBSPOT"]:
-            app.hubspot_api_key = config["HUBSPOT"]["API_KEY"]
-        # else:
-            #TODO throw error
+        try:
+            app.hubspot_access_token = config["HUBSPOT"]["ACCESS_TOKEN"]
```

Comment:
How long does the config["HUBSPOT"]["ACCESS_TOKEN"] last? Do we need to rotate it every x days? Or is it permanent until we decide to rotate it for security concerns?